python/morphology.cpp

```diff
@@ -13,6 +13,97 @@
 
 namespace pyarb {
 
+arb::segment_tree load_swc_allen(const std::string& fname, bool no_gaps=false) {
+        std::ifstream fid{fname};
+        if (!fid.good()) {
+            throw pyarb_error(util::pprintf("can't open file '{}'", fname));
+        }
+        try {
+            using namespace arb;
+            auto records = parse_swc_file(fid);
+            swc_canonicalize(records);
```

Comment:
this is redundant, see https://github.com/arbor-sim/arbor/blob/111104b4b6082a7768a6933cc54dbc065cffbd19/arbor/swcio.cpp#L148

Reply:
Indeed. We also canonicalize input in the normal `load_swc` function in C++/Python. I will fix that too.